build/Targets/RepoToolset/RepositoryInfo.targets

```diff
@@ -8,10 +8,12 @@
   <!--
     Include both GitHub and VSTS packages to enable SourceLink in repositories that mirror to VSTS.
   -->
+  <!-- DO NOT MERGE: enable linux build
```

Comment:
Why was this necessary? Is the sourcelink breaking for you and if so what was the error?